c2rust-ast-exporter/build.rs

```diff
@@ -181,7 +189,14 @@ fn build_native(llvm_info: &LLVMInfo) {
             "clangRewrite",
             "clangLex",
             "clangBasic",
-        ] {
+        ];
+        if llvm_info.llvm_major_version >= 15 {
+            // insert after clangSema
+            let sema_pos = clang_libs.iter().position(|&r| r == "clangSema").unwrap();
+            clang_libs.insert(sema_pos + 1, "clangSupport");
+        }
```

Comment:
I would put this inline as something like `if llvm_info.llvm_major_version >= 15 { "clangSupport" } else { "" }` and then filter out `""`.  It's much cleaner, easier to understand the order of libraries, and more extensible if we need to add conditional libraries where order matters.